wasmtime/_wasi.py

```diff
@@ -109,7 +110,7 @@ def __init__(self, store, name, config):
         self.__ptr__ = ptr
         self.store = store
 
-    def bind(self, import_):
+    def bind(self, import_: ImportType) -> typing.Optional["XInstance"]:
```

Comment:
Looks like `XInstance` is still mentioned here?

(how come this wasn't caught through testing?)

Reply:
Remember nothing is being checked at runtime by Python. And getting `typeguard`, the package we could use to do some run-time checking of some things, working with all the `import *` and so on was too much. I figured we could get `typeguard` on board once the `import *` have all gone away.